attributes/default.rb

```diff
@@ -10,7 +10,7 @@
 default['prometheus']['install_method']                                                   = 'source'
 
 # Prometheus version to build
-default['prometheus']['source']['version']                                                = '0.11.1'
+default['prometheus']['source']['version']                                                = '0.12.0'
```

Comment:
Line is too long. [100/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.